src/server/active_tran_server.cpp

```diff
@@ -69,31 +102,64 @@ int active_tran_server::init_page_server_hosts (const char *db_name)
       return ER_HOST_PORT_PARAMETER;
```

Comment:
":" is also searched in `parse_server_host ()`. this is a duplicate check. remove it?

Reply:
it is just a quick check before starting the parsing of parameter value, it can be removed with no impact on the functionality